spec/lib/s3_storage_spec.rb

```diff
@@ -157,7 +162,8 @@
 
   describe "#delete" do
     it "deletes the file from the S3 bucket" do
-      expect(s3_object).to receive(:delete).and_return(true)
+      allow(s3_object).to receive(:delete).and_return(true)
+      expect(s3_object).to receive(:delete)
       storage.delete(asset)
```

Comment:
Not a biggie and no need to change all of these, but these might read a little nicer with the syntax of:

```
      allow(s3_object).to receive(:delete).and_return(true)
      storage.delete(asset)
      expect(s3_object).to have_received(:delete)

```